src/components/recombination/common.rs

```diff
@@ -245,6 +272,7 @@ impl CycleCrossover {
         Box::new(Self::from_params(pc, insert_both))
     }
 
+    /// Creates a new `ArithmeticCrossover` which inserts only the first child.
```

Comment:
Typo: I think this should be cycle crossover.

Reply:
Fixed.